lib/addressable/uri.rb

```diff
@@ -1140,6 +1140,8 @@ def normalized_host
       # All normalized values should be UTF-8
       force_utf8_encoding_if_needed(@normalized_host)
       @normalized_host
+    rescue IDNA::Error => e
+      raise InvalidURIError.new(e)
```

Comment:
This is just an implicit `e.to_s` call, this could be clearer code
```suggestion
      raise InvalidURIError.new(e.message)
```

Reply:
Sure, I tend to like the implicit look as: "Look I'm just wrapping this exception" but no problem I just updated them for the explicit look in 7d75d6c0c27be22411870709cab3db5ae438eadc